app/src/main/java/io/intrepid/contest/screens/entrysubmission/entryname/EntryNamePresenter.java

```diff
@@ -18,24 +18,17 @@
     @Override
     public void onViewCreated() {
         super.onViewCreated();
-        view.setContestName("Chili Cookoff");
-    }
-
-    @Override
-    public void onEntryNameFocusChanged(boolean isFocused) {
-        if (isFocused) {
-            view.showEntryNameButton();
-        } else {
-            view.hideEntryNameButton();
-        }
+        view.showWelcomeMessage();
```

Comment:
Will this eventually take in a string? is that why we're making this its own function instead of just doing it in oncreate?